targets/PythonSdk/source/source/PlayFabSettings.py

```diff
@@ -0,0 +1,45 @@
+import PlayFabErrors
+import sys
+import traceback
+
+ProductionEnvironmentURL = "https://{titleId}.playfabapi.com{methodUrl}"
+TitleId = "" # You must set this value for PlayFabSdk to work properly (Found in the Game
+             # Manager for your title, at the PlayFab Website)
+DeveloperSecretKey = None # You must set this value for PlayFabSdk to work properly (Found in the Game
```

Comment:
The comment here isn't quite right.
This is only required for Admin/Server/Matchmaker to work correctly.